app/src/componentLibrary/systemMessage/systemMessage.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import informationImg from './img/information.png';
+import systemErrorImg from './img/SytemError.png';
+import warningImg from './img/warning.png';
+import styles from './systemMessage.scss';
+
+const cx = classNames.bind(styles);
+const MODE_INFORMATION = 'info';
+const MODE_WARNING = 'warning';
+const MODE_SYS_ERROR = 'error';
+let img = null;
+export const SystemMessage = ({ message, textSolution, children, mode }) => {
```

Comment:
According to the task component should have 2 states
1. width should be according to content
2. width should be 100% of parent container

Do you provide this behaviour?